jetbrains-rider/tst/base/AwsReuseSolutionTestBase.kt

```diff
@@ -38,10 +36,14 @@ abstract class AwsReuseSolutionTestBase : BaseTestWithSolutionBase() {
     override val testCaseNameToTempDir: String
         get() = getSolutionDirectoryName()
 
+    // TODO: Remove when https://youtrack.jetbrains.com/issue/RIDER-47995 is fixed FIX_WHEN_MIN_IS_203
```

Comment:
FYI, this is not public

Reply:
Agh. Sorry, this is an internal task that are internal by default. This is a ticket to handle VFS roots in Rider Test Framework in case we use local environment. I've made it visible for you. Please let me know if you prefer to delete the comment.

Reply:
No, I like the comment, but needed public so we could track this TODO

Reply:
Done.